soccer/MainWindow.cpp

```diff
@@ -949,6 +953,15 @@ void MainWindow::status(QString text, MainWindow::StatusType status) {
     }
 }
 
+void MainWindow::playStatus(bool color) {
```

Comment:
I don't like the name `playStatus` either, it feels a bit too generic. 

Can we name it something like `playChangedStatus` or `playIndicatorStatus` or similar?